src/compiler/driver.cc

```diff
@@ -125,18 +125,29 @@ Driver::~Driver() {}
 
 hilti::Result<hilti::Nothing> Driver::loadFile(hilti::rt::filesystem::path file,
                                                const hilti::rt::filesystem::path& relative_to) {
+    std::error_code ec;
     if ( ! relative_to.empty() && file.is_relative() ) {
-        if ( auto p = relative_to / file; hilti::rt::filesystem::exists(p) )
+        auto p = relative_to / file;
+        if ( auto exists = hilti::rt::filesystem::exists(p, ec); ! ec && exists )
             file = p;
+        if ( ec ) {
+            return hilti::rt::result::Error(
+                hilti::util::fmt("error computing path of %s relative to %s: %s", file, relative_to, ec.message()));
+        }
     }
```

Comment:
Here and also re/ the other changes below: I find this hard to read; suggest to generally change the pattern (1) execute operation, (2) check for error and potentially abort/return; and (3) proceed normally. So like:

```
auto exists = hilti::rt::filesystem::exists(p, ec);
if ( ec )
    return Error(...)

file = p;
...
```